src/utils/getLocalStorageItemKeys.ts

```diff
@@ -0,0 +1,11 @@
+const getLocalStorageItemKeys = (prefix: string) => {
+  const result = []
+  for (let i = 0; i < localStorage.length; i++) {
+    if (localStorage.key(i).startsWith(prefix)) {
```

Comment:
Could we check if the key exists before calling starWith ?

Reply:
Afaik calling Key function with index always return value. Should we add an additional check?

Reply:
Hmm. If that is the case, I think it's safe to keep it this way